lte/cloud/configs/nprobe.yml

```diff
@@ -10,8 +10,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# operator_id represents the mobile operator identifier
 # update_interval_secs sets the priodic time between runs in seconds.
 # backoff_interval_secs sets the backoff time when remote records collector is not
 # available.
+operator_id: 49002
```

Comment:
Will this always be a set to `49002` or is the expectation that it will be modified? If it can be modified we may want to look at including this in the terraform as a config variable.

Reply:
This is covered with other helm changes in this PR
https://github.com/magma/magma/pull/6515